.gitignore

```diff
@@ -11,3 +11,6 @@ tags
 .vscode
 .bsp
 .jekyll-cache/
+
+# metals
+metals.sbt
```

Comment:
💡 I can't remember who I've told, but ~/.config/git/ignore takes care of this on all your repos.